playbooks/rfcs.md

```diff
@@ -39,6 +39,24 @@ Find the right repo:
 - Does it affect the whole dev team? (use [README](https://github.com/artsy/README))
 - Can't figure it out or needs to be private? (use [Potential][] as a fallback)
 
+Create a plan for how the RFC will be implemented. If the change is complex or large in scope consider defining a
```

Comment:
I'd use "resolved" or "closed" here instead of "implemented" (as commented below), since the latter implies the RFC is adopted.

Reply:
My intent with this section was to address more complex, potentially multi-stage RFCs. E.g., if we're doing an RFC that has a trial run of something before we roll it out across more systems / the org, the implementation section could detail how the trial run will work and how the RFC might change based on the results. But I can also see that being part of the "approval" process as well.

Reply:
How do you envision making a larger / more complex change happen within the RFC system? E.g. maybe a series of smaller, more targeted RFCs that build off of each other as we work towards the intended outcome? I could imagine, for example, an RFC to align on the outcome we want and to do a trial, and then another RFC to write up trial results / feedback and propose a full rollout.